test/src/test/java/org/corfudb/recovery/FastObjectLoaderTest.java

```diff
@@ -558,29 +558,25 @@ public void ignoreStreamCheckpoint() throws Exception{
         populateMaps(1, getDefaultRuntime(), CorfuTable.class, false, 2);
 
         // Create a new runtime with fastloader
-        CorfuRuntime rt2 = getNewRuntime(getDefaultNode())
-                .connect();
+        CorfuRuntime rt2 = getNewRuntime(getDefaultNode()).connect();
         FastObjectLoader fsm = new FastObjectLoader(rt2);
         fsm.addStreamToIgnore("Map1");
         fsm.setDefaultObjectsType(CorfuTable.class);
         fsm.loadMaps();
 
         assertThatMapsAreBuiltIgnore(rt2, "Map1");
-
-
     }
 
     @Test(expected = RuntimeException.class)
-    public void failBecauseOfTrimAnd1Attempt() throws Exception {
+    public void failBecauseOfTrimAndZeroAttempts() throws Exception {
```

Comment:
If the lower test is exercising attempts = 0, should we leave this as the original? one attempt test.

Reply:
Agree, zero attempts sounds strange. 
Done